pkg/ca/ephemeralca/ephemeral_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ephemeralca
+
+import (
+	"crypto/x509"
+	"testing"
+	"time"
+
+	"github.com/sigstore/sigstore/pkg/cryptoutils"
+)
+
+func TestNewEphemeralCA(t *testing.T) {
+	ca, err := NewEphemeralCA()
+	if err != nil {
+		t.Fatalf("unexpected error generating ephemeral CA: %v", err)
+	}
+	if ca.RootCA == nil {
+		t.Fatalf("ca not set up")
+	}
+	if ca.RootCA.NotAfter.Sub(ca.RootCA.NotBefore) < time.Hour*87660 {
```

Comment:
if you spin this again, doing `time.Hour * 24 * 365 * 10` would be more readable

Reply:
done